### PR TITLE
Add documentation for -XX:[+|-]HandleSIGABRT

### PR DESCRIPTION
**Pull request by babsingh**
**Related:**
1. https://github.com/eclipse/openj9/issues/8735
2. https://github.com/eclipse/openj9/pull/9610

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
@babsingh ... Hi Babneet. Thanks for documenting this new option. Could you also add a short summary to the "What's New" topic for 0.21 (`version0.21.md`) that includes a link to 'xxhandlesigabrt.md'. You can see the general style and length required in previous "What's New" topics.

**Comment by babsingh** [comment]
@pmhayward `version0.21.md` has been updated in this pull request.

@DanHeidinga for review.

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/xxhandlesigabrt.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+-->
+
+# -XX:\[+|-\]HandleSIGABRT
+
+This option affects the handling of the operating system signal `SIGABRT`. This signal represents abnormal termination, and it can either be generated by the `abort` function or the `kill` command.
+
+
+## Syntax
+
+        -XX:[+|-]HandleSIGABRT
```
Comment (DanHeidinga) [inline docs/xxhandlesigabrt.md:32]: Spacing before the option intentional here?

File: docs/xxhandlesigabrt.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+|-----------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+HandleSIGABRT ` | Enable  | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+| `-XX:-HandleSIGABRT ` | Disable |                                                                                    |
+
+
+## Explanation
+
+When enabled, the VM handles the signal `SIGABRT` and generates the various dump files. 
+
+When the option is disabled, the VM does not handle the signal `SIGABRT`. Generally, this signal is handled by the default operating system handler.
```
[the remark this answers is not in the document]
Comment (babsingh) [reply docs/xxhandlesigabrt.md:44]: Yes, it applies to all operating systems openj9 supports. All have the SIGABRT signal. We always install the JVM abort handler.

File: docs/xxhandlesigabrt.md (hunk 1 above)
Comment (babsingh) [reply docs/xxhandlesigabrt.md:32]: I used [xxhandlesigxfsz.md](https://raw.githubusercontent.com/eclipse/openj9-docs/master/docs/xxhandlesigxfsz.md) as a base template for `xxhandlesigabrt.md`. All other markdown files have this spacing. The spacing is intentional to be consistent with other docs.

**Comment by DanHeidinga** [comment]
@pmhayward @SueChaplain any concerns with this being merged now?

**Pull request merged**